src/stories/Library/search-result-item/search-result-item.scss

```diff
@@ -8,7 +8,7 @@
   text-decoration: none;
 
   @include breakpoint-s {
-    height: 184px;
+    min-height: 184px;
```

Comment:
Please be aware of how this works in conjunction with #176 including the vertical layout this in Chromatic.